### ISSUE TITLE
Extend session timeout

### ISSUE THREAD

**Issue by lapega** [opensupports/opensupports#624]
The session timeout is too short, can you add an option in the configuration panel to set a greater value?

Thanks!

**Comment by BITJonas** [comment]
I second this. I love opensupports but this is nagging me. I would be happy if I could find the option in the database.

**Comment by mredigonda** [comment]
Yes, I agree, I'll create a PR for the next release. I think the default value is 24 minutes, and yes, that's way too short for a staff member if you happen to work on OpenSupports for 8 hours a day.

**Comment by mredigonda** [comment]
We are using session cookies to manage sessions, and they don't have an expiration date, they only expire when you close the browser, and according to [whitehadsec](https://www.whitehatsec.com/glossary/content/persistent-session-cookie), it will be a security risk to make them persistent.

**Comment by 7894008684das** [comment]
Hii Mr @mredigonda 

I am getting this error "You've no permission to access to this page"
while I am using localhost everything working fine but in the production environment after Login "You've no permission to access to this page" error coming.

for reference, I am attaching the screenshot. 
kindly help me.


![Screenshot (144)](https://user-images.githubusercontent.com/74444831/233791763-4a0d669c-9a2c-4963-992b-e1fbb48c399b.png)
![Screenshot (145)](https://user-images.githubusercontent.com/74444831/233791765-32ab0fd4-7478-413c-a90f-12ff85a8df4f.png)
![Screenshot (146)](https://user-images.githubusercontent.com/74444831/233791767-aca0007f-e763-471d-b288-cf78b6905f29.png)
